integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/TraceContextInjection.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.okhttp
+
+/**
+ * Defines whether the trace context should be injected into all requests or only sampled ones.
```

Comment:
should we explain what is the trace context and why it might be useful if request is not sampled (I, personally, have no idea, tbh).

Reply:
I just follow the same docs as in `iOS`. I am going to spend some time with Mael maybe we can define something better.